config/crd/bases/kepler.system.sustainable.computing.io_keplers.yaml

```diff
@@ -37,55 +36,11 @@ spec:
             properties:
               exporter:
                 properties:
-                  image:
-                    type: string
                   port:
                     default: 9103
                     type: integer
-                type: object
-              modelServer:
-                properties:
-                  path:
-                    default: ""
-                    type: string
-                  port:
-                    default: 8100
-                    type: integer
-                  promServer:
-                    default: ""
-                    type: string
-                  requiredPath:
-                    default: ""
-                    type: string
-                  trainer:
-                    properties:
-                      initialModelNames:
-                        default: ""
-                        type: string
-                      initialModelsEndpoint:
-                        default: ""
-                        type: string
-                      promHeaders:
-                        items:
-                          properties:
-                            headerKey:
-                              type: string
-                            headerValue:
-                              type: string
-                          type: object
-                        type: array
-                      promQueryInterval:
-                        default: 20
-                        type: integer
-                      promQueryStep:
-                        default: 3
-                        type: integer
-                      promSSLDisable:
-                        default: true
-                        type: boolean
-                    type: object
-                  url:
-                    default: ""
+                  version:
+                    default: latest
```

Comment:
I would suggest that we should use a stable version of Kepler as default since the latest might contain some developmental features

Reply:
Thats a todo before dev-preview is finalised

Reply:
@vprashar2929 it would nice if you could drive the validation in `kepler` so that certain images can be tagged `stable` and then we can rely on `stable` tag here.